### DISCUSSION TITLE
Welcome to ASFEnhance Discussions! 欢迎来到 ASFEnhance 讨论区!

### DISCUSSION THREAD

**Discussion by chr233** [chr233/ASFEnhance#16] [公告 / Announcement]
## 👋 Welcome!
 
If you find bugs or you want new features, you can post here

## 👋 欢迎!

如果遇到问题需要帮助或者想要添加新功能, 都可以在这里发帖

**Comment by ksardas2015** [comment]
First of all, I really want to thank the developer of the plugin. Very frequent and relevant updates, it's very cool :)
My name is Dima, I'm from Ukraine. I'm using ASF from november 2016 and recently found this awersome plugin. I think this plugin has a lot of potential, and in general I would like to talk with people who use ASF :)

**Reply by chr233** [reply]
thanks, I made this just for fun

**Comment by pontifezo** [comment]
ASFEnhance is great, very useful plugin!
Posted some wishes about additional functions here - https://github.com/chr233/ASFEnhance/discussions/18

**Comment by evgewakan** [comment]
thx for plugin!
